util.go

```diff
@@ -38,7 +38,7 @@ func encodeQuery(query map[string][]string) string {
 			continue
 		}
 
-		tuple := key + "=" + oDataEncodeVals(vals)
+		tuple := key + "=" + strings.Replace(strings.Join(vals, ","), " ", "%20", -1)
```

Comment:
Just realized that what oDataEncodeVals does is url.QueryEscape, which will probably mess those characters up. Still need @Page- to clarify if there's anything else we should do to sanitize the values.